config/registry_image_pruner/cronjob.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: image-pruner-cronjob
+spec:
+  schedule: "0 0 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: image-pruner
+            image: registry.redhat.io/rhel8/python-39
+            env:
+              - name: QUAY_TOKEN
+                valueFrom:
+                  secretKeyRef:
+                    name: quaytoken
+                    key: quaytoken
+              - name: NAMESPACE
+                valueFrom:
+                  secretKeyRef:
+                    name: quaytoken
+                    key: organization
+            imagePullPolicy: IfNotPresent
+            command:
+              - /bin/bash
+              - '-c'
+              - >-
+                pip install -r /image-pruner/requirements.txt && python
+                /image-pruner/prune_images.py --namespace $(NAMESPACE) --dry-run
```

Comment:
dry run not needed

Reply:
was still leaving that for testing purposes, but updated.